### ISSUE TITLE
Time shift with converting to pandas time series when time zone other than GMT

### ISSUE THREAD

**Issue by lgautier** [rpy2/rpy2#634]
```python
import rpy2.robjects as ro
x = ro.r('x <- as.POSIXct(1234567890, origin = "1960-01-01", tz = "UTC")') 
```

```python
>>>> print(x)
[1] "1999-02-13 23:31:30 UTC"
```

The convertion is appears off by 5 hours which would correspond to the difference between UTC and EST (my current timezone).

```python
import rpy2.robjects.pandas2ri as pd2r
```

```python
>>> pd2r.rpy2py_posixct(x)
DatetimeIndex(['1999-02-13 18:31:30+00:00'], dtype='datetime64[ns, UTC]', freq=None)
```

**Comment by lgautier** [comment]
In summary, the following is happening during the conversion step:

```python
import rpy2.robjects as ro
import datetime
import pytz

x = ro.r('x <- as.POSIXct(1234567890, origin = "1960-01-01", tz = "UTC")')
tzone_name = x.do_slot('tzone')[0]  # 'UTC'
tzone = pytz.timezone(tzone_name)

tzone.localize(datetime.datetime.fromtimestamp(x[0]))
# datetime.datetime(1999, 2, 13, 18, 31, 30, tzinfo=<UTC>)

datetime.datetime.fromtimestamp(x[0])
# datetime.datetime(1999, 2, 13, 18, 31, 30)
```

The call to `tzone.localize()` would have been expected to take a naive date time and perform the localization, but apparently it just adds a timezone attribute.

It looks like I misunderstood what it is doing.

**Comment by lgautier** [comment]
Probably cold comfort, but I am not the first one to fall for this:
https://stackoverflow.com/questions/1379740/pytz-localize-vs-datetime-replace

**Comment by lgautier** [comment]
A correct way to get the right timezone would seem to be:
```python
tzone.normalize(
    ro.vectors.get_timezone().localize(
        datetime.datetime.fromtimestamp(x[0])
    )
)
```

**Comment by lgautier** [comment]
Pull request in progress here:
https://github.com/rpy2/rpy2/pull/635

**Comment by lgautier** [comment]
There is an easier way to achieve this:
```python
datetime.datetime.fromtimestamp(x[0], tzone)
```

**Comment by lgautier** [comment]
PR #635 merged. Will be in included in release 3.2.6.
